tests/profiling/test_profiler.py

```diff
@@ -16,3 +19,30 @@ def test_restart():
     p.stop()
     p.start()
     p.stop()
+
+
+@pytest.mark.parametrize(
+    "service_name_var", ("DD_SERVICE", "DD_SERVICE_NAME", "DATADOG_SERVICE_NAME"),
+)
+def test_default_from_env(service_name_var, monkeypatch):
+    monkeypatch.setenv("DD_API_KEY", "foobar")
+    monkeypatch.setenv(service_name_var, "foobar")
+    prof = profiler.Profiler()
+    for exporter in prof.exporters:
+        if isinstance(exporter, http.PprofHTTPExporter):
+            assert exporter.service_name == "foobar"
+            break
+    else:
+        pytest.fail("Unable to find HTTP exporter")
+
+
+def test_service_api(monkeypatch):
+    monkeypatch.setenv("DD_API_KEY", "foobar")
+    prof = profiler.Profiler(service="foobar")
+    assert prof.service == "foobar"
+    for exporter in prof.exporters:
+        if isinstance(exporter, http.PprofHTTPExporter):
+            assert exporter.service_name == "foobar"
```

Comment:
Should we also do `service_name` -> `service` in the exporter?

Reply:
done